src/stateFilters.js

```diff
@@ -31,9 +31,28 @@ function $IncludedByStateFilter($state) {
     return $state.includes(state);
   };
   includesFilter.$stateful = true;
-  return  includesFilter;
+  return includesFilter;
+}
+
+/**
+ * @ngdoc filter
+ * @name ui.router.state.filter:includedByStateWithParams
+ *
+ * @requires ui.router.state.$state
+ *
+ * @description
+ * Translates to {@link ui.router.state.$state#methods_includes $state.includes('fullOrPartialStateName', {})}.
+ */
+$IncludedByStateWithParamsFilter.$inject = ['$state'];
```

Comment:
No reason for a whole new filter - you just can extend the original filter by adding a second `params` argument.